rdr_service/dao/participant_summary_dao.py

```diff
@@ -542,14 +548,27 @@ def _add_order_by_state(order_by, query):
         else:
             return query.order_by(Code.display.desc())
 
+    @classmethod
+    def _add_env_health_order(cls, query, order_by, field):
+        if not order_by.ascending:
+            field = field.desc()
+
+        return query.outerjoin(
+            PediatricDataLog,
+            and_(
+                PediatricDataLog.participant_id == ParticipantSummary.participantId,
+                PediatricDataLog.data_type == PediatricDataType.ENVIRONMENTAL_HEALTH
+            )
+        ).order_by(field)
+
     def _make_query(self, session, query_definition):
         query, order_by_field_names = super(ParticipantSummaryDao, self)._make_query(session, query_definition)
         # Note: leaving for future use if we go back to using a relationship to PatientStatus table.
         # query.options(selectinload(ParticipantSummary.patientStatus))
         # sql = self.query_to_text(query)
 
         if not query_definition.attributes:  # temporarily skip any joinloads if using result field filters
-            query.options(self._default_api_query_options())
+            query = query.options(*self._default_api_query_options())
```

Comment:
Was this not working before since the assignment the added assignment variable? Also can you explain the usage of the asterisk in with the parameter?

Reply:
Yeah, it wasn't working without the assignment. I missed testing the list endpoint functionality for this until looking for the env health fields.

`_default_api_query_options` returns a list, and the asterisk expands the list so that each element is passed in as a separate parameter. It looks like the `options` method can take a list and do the same anyway. So the change was mostly left over from me playing around with it and seeing whether there was a difference.

Reply:
Nice thanks for explaining.